llvm/utils/lit/tests/shtest-cat.py

```diff
@@ -0,0 +1,32 @@
+# Check the cat command
```

Comment:
```suggestion
## Test the cat command
```
Check can sometimes be easy to miss w/ `CHECK` directives, and we often use `##` to indicate that a line is commentary, and not intended to be part of the lit/FileCheck directives.

Reply:
done